### PR TITLE
[15163] Release notes Fast DDS v2.1.2 <2.1.x>

### PR DESCRIPTION
**Pull request by JLBuenoLopez-eProsima**
Signed-off-by: JLBuenoLopez-eProsima <joseluisbueno@eprosima.com>

### CODE REVIEW

File: docs/notes/previous_versions/v2.1.1.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Version 2.1.2
```
Comment (MiguelCompany) [inline docs/notes/previous_versions/v2.1.1.rst:1]:
```suggestion
Version 2.1.1
```

**Review by MiguelCompany — CHANGES_REQUESTED** [review]
Apart from my suggestion below, please bump version on conf.py

**Comment by JLBuenoLopez-eProsima** [comment]
CI is going to fail because this branch is not prepared to be built using CMake (#245 was introduced later on). Tests must be passed locally using `Makefile`: `make test`.

**Comment by JLBuenoLopez-eProsima** [comment]
@MiguelCompany I have applied your suggestions and fixed the spelling errors introduced by these changes. CI limitations are explained in the previous comment.

**Review by MiguelCompany — APPROVED** [review]
LGTM

**Pull request merged**